slike/fenko.tikz

```diff
@@ -0,0 +1,55 @@
+\begin{tikzpicture}[style=thick,scale=0.9]
+\small
+\tikzstyle{dec}=[draw,rectangle,minimum size=0.6cm]
+\tikzstyle{ran}=[draw,circle,minimum size=0.6cm]
+\tikzstyle{pick}=[double, very thick]
+\node[dec] (A) at (-8,-2) [label=left:$7.65$ G] {};
+\node[ran] (B) at (-4, 2) [label=135:$7.65$ G] {};
+\node[ran] (C) at (-4,-6) [label=225:$5.68$ G] {};
+\node      (D) at (-4,-2) {$5$ G};
+\node[dec] (E) at ( 0, 4) [label=225:$9$ G] {};
+\node[dec] (F) at ( 0, 0) [label=225:$3$ G] {};
+\node[dec] (G) at ( 0,-4) [label=225:$7$ G] {};
+\node[dec] (H) at ( 0, -8) [label=225:$3$ G] {};
+\node[ran] (I) at ( 4, 5) [label=225:$14.33$ G] {};
+\node      (J) at ( 4, 3) {$9$ G};
+\node[ran] (Y) at ( 4, 1) [label=225:$3.2$ G] {};
+\node      (K) at ( 4,-1) {$3$ G};
+\node[ran] (L) at ( 4,-3) [label=225:$7.6$ G] {};
+\node      (M) at ( 4,-5) {$7$ G};
+\node[ran] (N) at ( 4, -7) [label=225:$4.4$ G] {};
+\node      (O) at ( 4, -9) {$4$ G};
+\node      (P) at ( 8, 6) {$17$};
+\node      (R) at ( 8, 4) {$5$};
+\node      (S) at ( 8, 2) {$5$};
+\node      (T) at ( 8, 0) {$2$};
+\node      (U) at ( 8, -2) {$13$};
+\node      (V) at ( 8, -4) {$4$};
+\node      (Z) at ( 8, -6) {$8$};
+\node      (X) at ( 8, -8) {$2$};
+
+\draw[pick] (A) -- (B) node[above, midway, sloped] {kupi 2 kg};
+\draw (A) -- (C) node[below, midway, sloped] {kupi 1 kg};
+\draw (A) -- (D) node[below, midway, sloped] {kupi 0 kg};
+\draw (B) -- (E) node[above, midway, sloped] {uspe: $0.4$};
+\draw (B) -- (F) node[below, midway, sloped] {ne uspe: $0.6$};
+\draw (C) -- (G) node[above, midway, sloped] {uspe: $0.4$};
+\draw (C) -- (H) node[below, midway, sloped] {ne uspe: $0.6$};
+\draw[pick] (E) -- (I) node[above, midway, sloped] {kupi 4 kg};
+\draw (E) -- (J) node[below, midway, sloped] {kupi 0 kg};
```

Comment:
Načeloma so tukaj še vmesne možnosti (kupi 1, 2, 3 kg), vendar se strinjam, da bi bilo drevo v tem primeru nepregledno. Namesto tega v besedilu naloge raje utemelji, zakaj pride v upoštev samo nakup maksimalne količine granodiorita - v splošnem izračunaj E(*X* | *Y_2* = *k-2x*, *Z_2* = *x*), kjer je *k* število guldov po prvi noči in *x* število kilogramov kupljenega granodiorita pred drugo nočjo.